academic_observatory_workflows/workflows/crossref_metadata_telescope.py

```diff
@@ -276,7 +280,10 @@ def check_release_exists(self, **kwargs):
         url = CrossrefMetadataTelescope.TELESCOPE_URL.format(year=execution_date.year, month=execution_date.month)
         logging.info(f"Checking if available release exists for {execution_date.year}-{execution_date.month}")
 
-        response = retry_session().head(url)
+        # Get API key: it is required to check the head now
```

Comment:
could this use the api_key() method?

Reply:
Ah, not in this case because the release hasn't been created at this point

Reply:
ahh yeah, makes sense. Opened up the whole file (not just the diff) and went through it again. Thanks